### PR TITLE
[4.0] Adding language column to com_finder index backend

### PR DESCRIPTION
**Pull request by Hackwar**
The list of indexed items in the backend of com_finder does not have a column for the language. Since we have this everywhere else, I'm adding it here, too.

### CODE REVIEW

**Comment by Quy** [comment]
https://github.com/joomla/joomla-cms/pull/21038/files#diff-96ae0370a620b2bdb4168d9f9ebab161R67
`<td colspan="7">`

Update column to be `7` or `8` depending on `Multilanguage::isEnabled()`

**Comment by Hackwar** [comment]
Updated the colspan to 8. I think it would be a waste of CPU cycles to check here again if multilanguage or not. Since it doesn't matter if we have 7, 8, 9 or 42 here, as long as we want it to span the whole table, I wouldn't change this.

**Comment by infograf768** [comment]
Issue when filtering at the same time  content map and language
screen shot 2018-09-19 at 12 25 42

**Comment by infograf768** [comment]
I guess changes are needed in ContentmapField.php

**Comment by Hackwar** [comment]
There is no content that is both french and german at the same time, so this is an issue that we can't really fix. We could hide the language taxonomy branch when #21327 is also merged if it is a real issue.

**Comment by infograf768** [comment]
I think that we should not present at all in filters or elsewhere anything related to Content Languages when the languagefilter is disabled, as we now do everywhere in Joomla.

**Comment by Hackwar** [comment]
Taxonomy has no idea what a branch means and we could only introduce a special handling for a branch that is named "Language", which again could not be present at all or deactivated or something. I'm not a fan of that...

File: administrator/components/com_finder/Model/IndexModel.php [hunk 1]
```diff
@@ -211,6 +216,12 @@ protected function getListQuery()
 			$query->where($db->quoteName('l.published') . ' = ' . (int) $state);
 		}
 
+		// Filter on the language.
+		if ($language = $this->getState('filter.language'))
+		{
+			$query->where($db->quoteName('l.language') .' = ' . $db->quote($language));
```
Comment (Quy) [inline administrator/components/com_finder/Model/IndexModel.php]:
```suggestion
			$query->where($db->quoteName('l.language') . ' = ' . $db->quote($language));
```

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 77cbda15a88b2c644659746b00db96a9a7859dde<br /><br />I've tested this item with multilingual sample data and made a code inspection.From this point of view the PR works as described. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21038">issues.joomla.org/tracker/joomla-cms/21038</a>.</sub>

File: administrator/components/com_finder/tmpl/index/default.php [hunk 2]
```diff
@@ -85,6 +91,11 @@
 							<td class="small d-none d-md-table-cell text-center">
 								<?php echo JHtml::_('date', $item->indexdate, JText::_('DATE_FORMAT_LC4')); ?>
 							</td>
+							<?php if (Multilanguage::isEnabled()) : ?>
+								<td class="small d-none d-md-table-cell text-center">
+									<?php echo JLayoutHelper::render('joomla.content.language', $item); ?>
```
Comment (Quy) [inline administrator/components/com_finder/tmpl/index/default.php]:
```suggestion
									<?php echo LayoutHelper::render('joomla.content.language', $item); ?>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 52ece7144b38568033bd6571f9d4d6ea52d2b792<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21038">issues.joomla.org/tracker/joomla-cms/21038</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21038">issues.joomla.org/tracker/joomla-cms/21038</a>.</sub>

**Comment by Hackwar** [comment]
Can we finally merge this?

**Comment by roland-d** [comment]
Thank you.

**Comment by Hackwar** [comment]
Thank you!

**Pull request merged**